10_datastructures_lab/01.py

```diff
@@ -1,19 +1,8 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-
-When valid credentials are entered print:
-    Welcome Master
-
-And when invalid credentials are entered print:
-    INTRUDER ALERT
-
-and exit the program with a non-zero exit code
-"""
-
+"""get name and password and check if it correct"""
+import os,sys
+nep = {'apple' : "red" , 'lettuce' : "green" , 'lemon' : "yellow" , 'orange' : "orange"}
+for key in nep:
+    if sys.argv[1] == key and nep[key] == sys.argv[2] :
+        print "Welcome Master"
+        quit()

```

Comment:
Surprisingly, using `quit` is not a recommended way to quit a python program. Better to use:

```
import sys
sys.exit()
```

Or if you had an error:

```
import sys
sys.exit('something went wrong')
```